File: 21-01-20-pclassic/p3/StressfulPark.java
import java.io.File;
import java.io.FileNotFoundException;
import java.util.Scanner;

public class StressfulPark {

  public static void main(String[] args) throws FileNotFoundException {

    Scanner fileInput = new Scanner(new File("StressfulParkIN.txt"));

    while (fileInput.hasNext()) {

      String inputStr = fileInput.nextLine().trim();
      String[] inputStrArr = inputStr.split(" ");
      int N = Integer.parseInt(inputStrArr[0]);
      int M = Integer.parseInt(inputStrArr[1]);

      int[][] parkGrid = new int[N][M];

      for (int i = 0; i < N; i++) {
        String[] nextRow = fileInput.nextLine().trim().split(" ");

        for (int j = 0; j < M; j++) {
          parkGrid[i][j] = Integer.parseInt(nextRow[j]);
        }
      }

      System.out.println(shortestTimeToExit(parkGrid));

    }
    fileInput.close();

  }

  public static int shortestTimeToExit(int[][] parkGrid) {
    // Your implementation here!
    return 0;
  }

}

File: 21-01-20-pclassic/p3/StressfulPark.py
def path(park) -> int:
    # Your implementation here!
    return 0


if __name__ == '__main__':
    with open('StressfulParkIN.txt', 'r') as f:
        while True:
            s = f.readline()
            if s == '':
                break
            N, M = map(int, s.strip().split())
            park = []
            for i in range(N):
                park.append(list(map(int, f.readline().strip().split())))
            print(path(park))
